weblate/checks/placeholders.py

```diff
@@ -4,9 +4,9 @@
 
 from __future__ import annotations
 
-import re
 from typing import TYPE_CHECKING, Any, Literal
 
+import regex as re
```

Comment:
I'd prefer to use the original name to make it clear that a different implementation is in use.

```suggestion
import regex
```